src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -148,6 +149,8 @@ public Object instanceMainPostParseArgs() {
             BlockGunzipper.setDefaultInflaterFactory(new IntelInflaterFactory());
         }
 
+        BucketUtils.setGlobalNIODefaultOptions();
```

Comment:
Should we move this into instanceMain before arg parsing?  That way when we update barclay to use paths it will benefit from this.